### PR TITLE
Add Inception V4, EfficientNet and EfficientNet Lite models

### PR DESCRIPTION
**Pull request by mariecwhite**

### CODE REVIEW

File: tflitehub/efficientnet_lite0_int8_test.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    super(EfficientnetLite0Int8Test, self).__init__(model_path, *args, **kwargs)
+
+  def compare_results(self, iree_results, tflite_results, details):
+    super(EfficientnetLite0Int8Test, self).compare_results(iree_results, tflite_results, details)
+    # Dequantize outputs.
+    zero_point = details[0]['quantization_parameters']['zero_points'][0]
+    scale = details[0]['quantization_parameters']['scales'][0]
+    dequantized_iree_results = (iree_results - zero_point) * scale
+    dequantized_tflite_results = (tflite_results - zero_point) * scale
+    self.assertTrue(numpy.isclose(dequantized_iree_results, dequantized_tflite_results, atol=5e-3).all())
```
Comment (jpienaar) [inline tflitehub/efficientnet_lite0_int8_test.py:22]: OOC how are these bounds determined?

File: tflitehub/inception_v4_uint8_test.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# RUN: %PYTHON %s
+
+import absl.testing
+import imagenet_test_data
+import numpy
+import test_util
+
+# Source https://tfhub.dev/tensorflow/lite-model/inception_v4_quant/1/default/1
```
Comment (jpienaar) [inline tflitehub/inception_v4_uint8_test.py:8]: Should we perhaps use inception_v4_quant in the name of the test file to make tracking back to tfhub easier?

**Review by jpienaar — COMMENTED** [review]
Nice! (drive by random questions)

File: tflitehub/efficientnet_lite0_int8_test.py (hunk 1 above)
Comment (mariecwhite) [reply tflitehub/efficientnet_lite0_int8_test.py:22]: They are metadata that's part of the tensor. The values are calculated during training - a running count of the min/max values are kept for each tensor and during the conversion step from Tensorflow to TFLite, these min/max values are used to convert the tensor values from FP32 to UINT8 (or INT8). Quantization parameters 'zero_point' and 'scale' are kept around in order to convert the quantized values back to float. For details: https://arxiv.org/abs/1712.05877 (or ask Benoit Jacob who authored the paper :)

File: tflitehub/inception_v4_uint8_test.py (hunk 2 above)
Comment (mariecwhite) [reply tflitehub/inception_v4_uint8_test.py:8]: It's useful to differentiate between UINT8 and INT8 models, which the tfhub naming unfortunately doesn't do. For example, that accuracy regression that we just ran into only occurred in INT8 models and it was really easy to tell by looking at the test name.
